src/ci/docker/cross/Dockerfile

```diff
@@ -19,7 +19,13 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   bzip2 \
   patch \
   libssl-dev \
-  pkg-config
+  pkg-config \
+  swig \
+  libedit-dev \
+  libncurses5-dev
+
+RUN curl -L https://cmake.org/files/v3.8/cmake-3.8.0-rc1-Linux-x86_64.tar.gz | \
```

Comment:
Could you throw in a small comment to why a newer version of CMake is needed?

Also if switching to ubuntu 16.10 pulls in a new enough version it'd be ok to update to that as well.